app.py

```diff
@@ -7,15 +7,25 @@
     app.run()
 
 
+@app.route('/test', methods=['POST'])
+def test_path2():
+    return True
+
+
+@app.route('/test')
+def test_path():
+    return "test path"
+
+
 @app.route('/')
 def hello_world():
     conn = sqlite3.connect('data_base.db')
     curs = conn.cursor()
     curs.execute('''CREATE TABLE IF NOT EXISTS users (
-                    Id INTEGER PRIMARY KEY AUTOINCREMENT,
-                    Username TEXT NOT NULL UNIQUE,
-                    Password TEXT NOT NULL
-                    )''')
+                       Id INTEGER PRIMARY KEY AUTOINCREMENT,
```

Comment:
Why did you add these spaces? seem unnecessary.

Reply:
if you are talking about lines 14 -18: than it's just spacing, since the query is fairly long.
is it not standard for us to space long queries?